lib/PHPExif/Adapter/Exiftool.php

```diff
@@ -100,8 +101,14 @@ public function setEncoding(array $encodings) : void
     public function getToolPath() : string
     {
         if ($this->toolPath === '') {
-            // Do not use "which": not available on sh
-            $path = exec('command -v ' . self::TOOL_NAME);
+            try {
+                // Do not use "which": not available on sh
+                $path = exec('command -v ' . self::TOOL_NAME);
+            } catch (ExecException) {
+                // @codeCoverageIgnoreStart
+                $path = false;
+                // @codeCoverageIgnoreEnd
+            }
```

Comment:
This can be simplified with the code below. Below an `if` tests for `$path !== false`. This isn't necessary, if the code from the `if` below would be put into the `try`-block. This would also removed the need for `$path = false` in the `catch`-block.

Reply:
Fixed : [ccc6fe1](https://github.com/LycheeOrg/php-exif/pull/48/commits/ccc6fe1aa1abdd207371f75fbb241a10fdf2af4a)